src/Illuminate/Http/Concerns/InteractsWithInput.php

```diff
@@ -309,9 +309,7 @@ public function allFiles()
     {
         $files = $this->files->all();
 
-        return $this->convertedFiles
-                    ? $this->convertedFiles
-                    : $this->convertedFiles = $this->convertUploadedFiles($files);
+        return $this->convertedFiles = $this->convertedFiles ?? $this->convertUploadedFiles($files);
```

Comment:
Could this not be shortened further to just `return $this->convertedFiles ?? $this->convertUploadedFiles($files);`?

Reply:
No, we also want to set the value for $this->convertedFiles when it is falsy before returning.

Reply:
@richardkeep Gotcha 🙂